src/components/textarea/textarea.stories.mdx

```diff
@@ -82,6 +83,9 @@ import Textarea from "carbon-react/lib/components/textarea";
 
 ### With characterLimit
 
+If you use the `inputHint` prop to provide the user with a hint before the input, please use a full stop `.` at the end,
+as it forces a pause before any other announcements, this well help screan reader users understand the hint fully.
```

Comment:
```suggestion
as it forces a pause before any other announcements, this well help screen reader users understand the hint fully.
```

Reply:
Brilliant spot 👍, blaming the new keyboard for this one :/